tests/GenFu.Tests/GenFuTests.cs

```diff
@@ -447,5 +447,43 @@ public void MethodCanBeSet()
             Assert.True(!string.IsNullOrEmpty(person.GetMiddleName()));
             Assert.Equal(expected, person.GetMiddleName());
         }
+
+        [Fact]
+        public void NullableDateWillGetRandomNullSeed()
+        {
+
+            var dates = new List<DateTime?>();
+            for(int x = 0; x<100; x++)
+            {
+                A.Reset();
+                var person = A.New(new Person());
+                dates.Add(person.DateOfDeath);
+            }
+
+            int nullItems = dates.Where(x => x == null).Count();
+            int notNullItems = dates.Where(x => x != null).Count();
+
+            // Should contain mix of Nulls and set values
+            Assert.True(nullItems > 0);
+            Assert.True(notNullItems > 0);
+        }
+
+        [Fact(Skip ="Not yet supported but should be")]
+        public void IntIsPopulatedIfNullable()
+        {
+            A.Reset();
+            var person = A.New(new Person());
+
+            Assert.NotEqual(null, person.NumberOfCats);
+        }
+
+        [Fact]
+        public void StringIsPopulatedIfNullable()
+        {
+            A.Reset();
+            var person = A.New(new Person() { FirstName = null } );
+
+            Assert.NotNull(person.FirstName);
```

Comment:
The seed is designed to allow the caller/consumer to determine how many items within the collection will be Null. for example : Min=2, Max: 10 will make (on average) 20% of the list null. This is handy where you might want to simulate a system where you know that only 5% of data has null values; or 95% of data has null values. i have updated by Repos to include a test that shows how this works.
If you set the seed to min:0 and max:10 will make the property Null every-time. You can also make sure properties are null with a Configure.Fill statement??